data/map planets.txt

```diff
@@ -43,6 +43,13 @@ planet Ada
 		threshold 8000
 		fleet "Large Republic" 16
 
+planet "Aetuah"
+	attributes "gas giant" "requires: gaslining"
+	landscape land/nasa30
+	description `The close proximity of Aetuh and the neighboring ice moon causes various ebbs and flows of the storms on the gas giant as well as constant reshaping of the moon's surface. The ice moon may have been captured by Aetuh in times past as a rogue planet, and the distance of the gas giant from the system center suggests it might be a rogue planet itself as well. In time, the gravitational force exhibited by Aetuah will tear apart the ice moon, plummeting it into the abyss.`
```

Comment:
```suggestion
	description `The close proximity of Aetuh and the neighboring ice moon causes various ebbs and flows in the storms on the gas giant as well as constant reshaping of the moon's surface. The ice moon may have been captured by Aetuh in times past as a rogue planet, and the distance of the gas giant from the system center suggests it might be a rogue planet itself as well. In time, the gravitational force exhibited by Aetuah will tear apart the ice moon, plummeting it into the abyss.`
```